### PR TITLE
Minor formatting changes

### PR DESCRIPTION
**Pull request by deshankoswatte**
- Re initiation of https://github.com/yashodgayashan/environment_platform/pull/18 with most of the suggestions added.
- Added TODOs for return values and debug logs, this is due to the fact that there are some problems with MongoDB client and it shows errors in my editor.

### CODE REVIEW

File: core/src/env_platform/db_handler.bal [hunk 1]
```diff
@@ -73,10 +53,45 @@ function saveApplication(TreeRemovalForm form) returns error? {
...
+# array index out of bound if there are no applications with the specific application Id.
+function deleteApplication(string applicationId) returns boolean|error {
+
+    // TODO - Return true or false based on deleted.
+    string applicationStatus = check getApplicationStatusByApplicationId(applicationId);
+    if (applicationStatus == "draft") {
+        int deleted = check applicationCollection->delete({"applicationId": applicationId, "status": "draft"});
+        return true;
+    } else {
+        return error("Invalid Operation", message = "Cannot delete the application with the appilcationId: "
```
Comment (sanjulamadurapperuma) [inline core/src/env_platform/db_handler.bal:77]:
```suggestion
        return error("Invalid Operation", message = "Cannot delete the application with the application ID: "
```

**Comment by deshankoswatte** [comment]
@yashodgayashan or @vikumkbv?

**Comment by deshankoswatte** [comment]
Fixed TODOs for return values for methods `saveApplication` and `deleteApplication`.

File: core/src/env_platform/db_handler.bal [hunk 2]
```diff
@@ -58,7 +57,10 @@ function saveApplication(TreeRemovalForm form) returns error? {
                 }
             ]
     };
-    return applicationCollection->insert(application);
+
+    mongodb:DatabaseError? insertted = applicationCollection->insert(application);
```
Comment (sanjulamadurapperuma) [inline core/src/env_platform/db_handler.bal:61]:
```suggestion
    mongodb:DatabaseError? inserted = applicationCollection->insert(application);
```

File: core/src/env_platform/db_handler.bal [hunk 3]
```diff
@@ -58,7 +57,10 @@ function saveApplication(TreeRemovalForm form) returns error? {
...
+
+    return insertted is mongodb:DatabaseError ? insertted : true;
```
Comment (sanjulamadurapperuma) [inline core/src/env_platform/db_handler.bal:63]:
```suggestion
    return inserted is mongodb:DatabaseError ? inserted : true;
```

**Comment by yashodgayashan** [comment]
> **Fixed TODOs for return values of methods** `saveApplication` and `deleteApplication`. Now these method will wither return a boolean or an error. Only debug logs are now required I think we should discuss about that offline first and then put another PR for that.

Looks good to me.

**Comment by yashodgayashan** [comment]
@sanjulamadurapperuma I have committed requested changes

**Comment by deshankoswatte** [comment]
@sanjulamadurapperuma Please approve.

**Pull request merged**